### PR TITLE
Add Windows CI pipeline for TensorRT

### PR DESCRIPTION
**Pull request by stevenlix**
-Add Windows CI pipeline
-Enable some onnx unit tests for TensorRT
-Fix minor issues in the code

### CODE REVIEW

File: tools/ci_build/build.py [hunk 1]
```diff
@@ -700,8 +700,9 @@ def main():
             if is_windows() or not os.path.exists(onnx_test_data_dir):
                 onnx_test_data_dir = os.path.join(source_dir, "cmake", "external", "onnx", "onnx", "backend", "test", "data")
             if args.use_tensorrt:
-              # Disable onnx unit tests for TensorRT because many tests are not supported yet
-              run_onnx_tests(build_dir, configs, '', 'tensorrt', False, 1)
+              # Disable some onnx unit tests that TensorRT parser doesn't supported yet
+              onnx_test_data_dir = os.path.join(source_dir, "cmake", "external", "onnx", "onnx", "backend", "test", "data", "simple")
```
Comment (jywu-msft) [inline tools/ci_build/build.py]: what is the purpose of line 704?  (test_data_dir points to simple sub directory)

File: tools/ci_build/github/azure-pipelines/win-gpu-tensorrt-ci-pipeline.yml [hunk 2]
```diff
@@ -111,7 +115,7 @@ jobs:
...
+        arguments: '$(Build.SourcesDirectory)\tools\ci_build\build.py --config Release --build_dir $(Build.BinariesDirectory) --cmake_path $(Build.BinariesDirectory)\cmake\bin\cmake.exe --ctest_path $(Build.BinariesDirectory)\cmake\bin\ctest.exe  --enable_pybind --use_openmp --use_mkldnn --build_shared_lib  --enable_onnx_tests --use_cuda --cuda_home="C:\local\cuda_10.0.130_win10" --cudnn_home="C:\local\cudnn-10.0-windows10-x64-v7.3.1.20\cuda" --use_full_protobuf --use_tensorrt --tensorrt_home="C:\local\TensorRT-5.0.4.3" --test_data_url $(TestDataUrl) --test_data_checksum $(TestDataChecksum) --test'
```
Comment (jywu-msft) [inline tools/ci_build/github/azure-pipelines/win-gpu-tensorrt-ci-pipeline.yml]: don't need both --use_tensorrt and --use_cuda

File: tools/ci_build/build.py (hunk 1 above)
Comment (stevenlix) [reply tools/ci_build/build.py]: To enable onnx unit tests. For debug mode those tests must be enabled.

File: tools/ci_build/github/azure-pipelines/win-gpu-tensorrt-ci-pipeline.yml [hunk 3]
```diff
@@ -75,7 +79,7 @@ jobs:
...
+        arguments: '$(Build.SourcesDirectory)\tools\ci_build\build.py --config Debug --build_dir $(Build.BinariesDirectory) --cmake_path $(Build.BinariesDirectory)\cmake\bin\cmake.exe --ctest_path $(Build.BinariesDirectory)\cmake\bin\ctest.exe  --enable_pybind --use_openmp --use_mkldnn --use_mkldnn --build_shared_lib  --enable_onnx_tests --use_cuda --cuda_home="C:\local\cuda_10.0.130_win10" --cudnn_home="C:\local\cudnn-10.0-windows10-x64-v7.3.1.20\cuda" --use_full_protobuf --use_tensorrt --tensorrt_home="C:\local\TensorRT-5.0.4.3" --test_data_url $(TestDataUrl) --test_data_checksum $(TestDataChecksum) --test'
```
Comment (jywu-msft) [inline tools/ci_build/github/azure-pipelines/win-gpu-tensorrt-ci-pipeline.yml]: don't need both --use_tensorrt and --use_cuda

File: tools/ci_build/github/azure-pipelines/win-gpu-tensorrt-ci-pipeline.yml [hunk 4]
```diff
@@ -58,7 +62,7 @@ jobs:
...
+        arguments: '$(Build.SourcesDirectory)\tools\ci_build\build.py --config Debug Release --build_dir $(Build.BinariesDirectory) --cmake_path $(Build.BinariesDirectory)\cmake\bin\cmake.exe --ctest_path $(Build.BinariesDirectory)\cmake\bin\ctest.exe  --enable_pybind --use_openmp --use_mkldnn --use_mkldnn --build_shared_lib  --enable_onnx_tests --use_cuda --cuda_home="C:\local\cuda_10.0.130_win10" --cudnn_home="C:\local\cudnn-10.0-windows10-x64-v7.3.1.20\cuda" --use_full_protobuf --use_tensorrt --tensorrt_home="C:\local\TensorRT-5.0.4.3" --test_data_url $(TestDataUrl) --test_data_checksum $(TestDataChecksum) --update --msvc_toolset=14.11'
```
Comment (jywu-msft) [inline tools/ci_build/github/azure-pipelines/win-gpu-tensorrt-ci-pipeline.yml]: don't need both --use_tensorrt and --use_cuda

Comment (stevenlix) [reply tools/ci_build/github/azure-pipelines/win-gpu-tensorrt-ci-pipeline.yml]: removed --use_cuda

File: tools/ci_build/build.py (hunk 1 above)
Comment (jywu-msft) [reply tools/ci_build/build.py]: so only the tests in the 'simple' subdir currently passes and nothing else?

Comment (stevenlix) [reply tools/ci_build/build.py]: Yes. Not all tests in other folders are supported by TensorRT

**Pull request merged**